plugin/modelgen/models_test.go

```diff
@@ -42,4 +45,30 @@ func TestModelGeneration(t *testing.T) {
 			require.True(t, len(words) > 1, "expected description %q to have more than one word", text)
 		}
 	})
+
+	t.Run("tags are applied", func(t *testing.T) {
+		node, err := parser.ParseFile(token.NewFileSet(), "./out/generated.go", nil, 0)
+		require.NoError(t, err)
+		for _, obj := range node.Scope.Objects {
```

Comment:
holy moly batman.

would this be good enough (and a lot more understandable) if we just searched for the expected string in the file?

Reply:
Yea, I guess that's enough and a lot simpler. Changed it.